samples/NHSD.BuyingCatalogue.Identity.Api.SampleMvcClient/Controllers/HomeController.cs

```diff
@@ -44,10 +44,9 @@ public async Task<IActionResult> Privacy()
             }
             else
             {
-                var content = await response.Content.ReadAsStringAsync();
-                Console.WriteLine(JArray.Parse(content));
-                ViewBag.Response = JArray.Parse(content);
+                ViewBag.Response = "Authorized With Sample Resource";
             }
+            Console.WriteLine(ViewBag.Response);
```

Comment:
Could you review this line please? I think we don't need this extra logging here unless it is used for something I am missing.

Reply:
Yeah we probably don't need it, I was just tidying up the pre-existing Console.WriteLine's to be simpler, instead of it appearing inside both the If and the else. Can strip it out for sure!